### PR TITLE
Allow everyone with edit permissions to delete episodes

### PR DESCRIPTION
**Pull request by chrisrhoden**

### CODE REVIEW

File: test/policies/episode_policy_test.rb [hunk 1]
```diff
@@ -57,5 +62,17 @@ def token(scopes, set_account_id=account_id)
...
     end
+
+    it 'allows deleting a draft episode' do
+      episode = create(:episode, published_at: nil, podcast: podcast)
+
+      assert EpisodePolicy.new(draft_token, episode).destroy?
+    end
+
+    it 'does not allow deleting a non-draft episode' do
+      episode = create(:episode, published_at: 25.years.ago, podcast: create(:podcast, prx_account_uri: "/api/v1/accounts/#{account_id}"))
```
Comment (cavis) [inline test/policies/episode_policy_test.rb:73]: Why the create-podcast here, instead of using the build_stubbed'd one?

Comment (chrisrhoden) [reply test/policies/episode_policy_test.rb:73]: Tests were giving me guff about build_stubbed touching the database

**Comment by cavis** [comment]
(Though it has a merge conflict)

**Comment by chrisrhoden** [comment]
@cavis fixed!

**Pull request merged**